priv/posts/marketing/20240530023712_getting-to-kow-natalia-chechina.md

```diff
@@ -0,0 +1,85 @@
+
+#### Which are the challenges and advantages of working with Erlang and Erlang VM in embedded systems? and what are the differences working with other languages?
+
+I will answer in relation to robotics because that's what I specialize in within Embedded systems. At the moment robotics doesn’t utilize distributed systems’ state-of-the-art to the extent that we used to in the BEAM community. So, robotics can hugely benefit from the knowledge, techniques, and approaches accumulated by the BEAM community. To me the main challenge for Erlang to be accepted as a tool by the roboticits is the communication barrier and relative nicheness of the BEAM languages. These make the entry point for developers into the robotics community very steep. The majority of people who work with robots are not developers. Their disciplines have developed tools and language to describe the abilities and limitations of robots. The problem is that the words we use in the BEAM community and the words that roboticists use are very similar, but have very different meanings. 
+
+So it’s very difficult to communicate where problems are and how Erlang can address existing issues. Simple examples are “safety” and “fault-tolerance”. Usually, when we talk about safety it’s information protection, while roboticists mean mechanisms and guarantees to protect humans from being harmed by robots. By fault-tolerance we usually mean an ability to recover from failures automatically; while for roboticists it’s an ability of a robot to continue with their tasks while some parts are not working. 
+
+
+So, when we say that a robot has safety issues or is not fault-tolerant, roboticists assume that we are completely incompetent in the area. It takes some significant effort and stubbornness to figure out the differences and start communicating so that roboticists start listening to the ideas we want to bring to the table. Another issue is that roboticists in general are not interested in learning a new language like Erlang or Elixir. So, apart from developing an idea, we need to make sure that there is a simple API and compatibility with existing tools. This leads to a very steep entrance to the robotics community. 
+
+#### Are there any future projects or gold that you want to share with us?
+
+There are lots of exciting bits and bobs but I don’t think I have a gold project at the moment… I’m supporting Erlang Workshop co-chairs in organising the workshop – it’s going to be in Milan, Italy this year. So, if you happen to be in Italy/Europe in September, do consider attending the Workshop – it’s always an exciting place to be to learn about new research and ideas in Functional programming.
+
+In the Embedded Systems WP we work on developing a bank of projects for developers and academics, so that these can be picked up when developers/academics have time and would like to work on a project that is of value to the community.

```

Comment:
```suggestion
In the Embedded Systems WG we work on developing a bank of projects for developers and academics, so that these can be picked up when developers/academics have time and would like to work on a project that is of value to the community.

```